src/services/item/plugins/publication/validation/utils.ts

```diff
@@ -1,9 +1,37 @@
 import path from 'path';
 import striptags from 'striptags';
 
+import { ItemType, MimeTypes } from '@graasp/sdk';
+
 import { TMP_FOLDER } from '../../../../../utils/config';
+import { FolderItem, Item, isItemType } from '../../../entities/Item';
 
 export const stripHtml = (str?: string | null): string => (str ? striptags(str) : '');
 
 export const buildStoragePath = (itemId: string): string =>
   path.join(TMP_FOLDER, 'validations', itemId);
+
+export const assertItemIsFolder = (item: Item): FolderItem => {
+  if (item.type !== ItemType.FOLDER) {
+    throw new Error();
+  }
+
+  return item as FolderItem;
+};
+
+export const isFileType = (item: Item) => {
+  return isItemType(item, ItemType.S3_FILE) || isItemType(item, ItemType.LOCAL_FILE);
+};
+
+export const getMimeType = (item: Item<'s3File'> | Item<'file'>) => {
+  const { mimetype } = item.type === ItemType.S3_FILE ? item.extra.s3File : item.extra.file;
+  return mimetype;
+};
```

Comment:
It is only a suggestion, I'm not sure if it is conventional or not. We're probably losing type safety ?
 
```suggestion
export const getMimeType = (item: Item<'s3File'> | Item<'file'>) => {
  const { mimetype } = item.extra[item.type];
  return mimetype;
};
```

Reply:
I really like the idea, but we are losing the type, so I think I'll let like it is.